### PR TITLE
Add __eq__ method and tests

### PR DESCRIPTION
**Pull request by erikmannerfelt**
Please see #113 for the motivation.

### CODE REVIEW

File: geoutils/georaster.py [hunk 1]
```diff
@@ -203,6 +203,20 @@ def __str__(self):
         """ Provide string of information about Raster. """
         return self.info()
 
+    def __eq__(self, other) -> bool:
```
Comment (rhugonnet) [inline geoutils/georaster.py:206]: so this would mean the data + CRS are equal, but not necessarily on disk (could be a copy) ?!

Comment (rhugonnet) [reply geoutils/georaster.py:206]: Nvm I see this in your description now

**Review by rhugonnet — APPROVED** [review]
Looks good to me :)

**Comment by adehecq** [comment]
Nice! Looks good to me too.

**Comment by rhugonnet** [comment]
All good, merging (as I'm not sure @erikmannerfelt has the rights yet)

**Comment by rhugonnet** [comment]
Resolves #113

**Comment by atedstone** [comment]
Starts to resolve #73

**Pull request merged**